migrations/2023-08-13-175220_to_semver_no_prerelease_parallel_safe/up.sql

```diff
@@ -0,0 +1,17 @@
+DROP FUNCTION to_semver_no_prerelease(text);
+DROP TYPE semver_triple;
+
+CREATE TYPE semver_triple AS (
+  major numeric,
+  minor numeric,
+  teeny numeric
+);
```

Comment:
Do we need to recreate the type?

Reply:
No reason in particular. I copied it from a different migration that did that. Now it's fixed.